script.tag-generator/main.py

```diff
@@ -0,0 +1,643 @@
+from bs4 import BeautifulSoup
+import datetime
+import json
+import os
+import re
+import requests
+import simplejson
+import sys
+import xbmc
+import xbmcgui
+import xbmcaddon
+
+sys.path.append(os.path.join(os.path.abspath(os.path.dirname(__file__)), "resources", "lib"))
```

Comment:
This is really a bad practice. Why is it needed? Can't you just import your modules from.resources.lib?

Reply:
I don't like it either, but if I don't import it this way the trakt module doesn't work. I've tried:
```
import resources.lib.trakt as trakt
from trakt import users as traktusers
```
and
```
import resources.lib.trakt as trakt
from resources.lib.trakt import users as traktusers
```
The latter results in an error in the trakt users module, which isn't written by me. I prefer hacky to editing someone else's code.